sitemap/sitemap.ts

```diff
@@ -0,0 +1,84 @@
+import { merge } from "lume/core/utils.ts";
+import { Page } from "lume/core/filesystem.ts";
+import { buildFilter, buildSort } from "lume/plugins/search.ts";
+
+import type { Site } from "lume/core.ts";
+
+export interface Options {
+  /** The filters to search pages included in the sitemap */
+  filters: string[];
+
+  /** The values to sort the sitemap */
+  sort: string[];
+
+  /** The default filters including the index page (only if custom `filters` are defined) */
+  defaultFilters: string[];
+
+  /** Set `true` append your filters to the defaults (only if custom `filters` are defined) */
+  keepDefaultFilters: boolean;
```

Comment:
This is not needed because `filters` (renamed to `query`, for consistency with the `search.pages()` arguments) should be empty by default.

Reply:
Thanks for the info.